src/CSModule.sol

```diff
@@ -761,22 +761,23 @@ contract CSModule is
                 penaltyMultiplier
             );
 
-            bool chargeWithdrawalRequestFee;
+            ICSAccounting _accounting = accounting();
+            bool chargeWithdrawalRequestFee = false;
+            bool bondCoveredCharges = true;
 
-            // It is safe to use unchecked for penalty sum because base penalties and fees are limited to uint248 in the
-            // MarkedUint248 structures used to store them, and the maximum multiplier is limited to 64, so
-            // `type(uint248).max * 64 * 2 < type(uint256).max`.
             ExitPenaltyInfo memory exitPenaltyInfo = EXIT_PENALTIES
                 .getExitPenaltyInfo(withdrawalInfo.nodeOperatorId, pubkey);
             if (exitPenaltyInfo.delayPenalty.isValue) {
-                unchecked {
-                    penaltySum +=
-                        exitPenaltyInfo.delayPenalty.value *
-                        penaltyMultiplier;
-                }
+                bondCoveredCharges = _accounting.chargeFee(
```

Comment:
Can we have one call of `_accounting.chargeFee` instead of 2 in the current form?